samples/dnn/models.yml

```diff
@@ -15,6 +15,30 @@ opencv_fd:
   rgb: false
   sample: "object_detection"
 
+# YOLO object detection family from Darknet (https://github.com/AlexeyAB/darknet)
+# Might be used for YOLOv4 and TinyYolov4
+yolov4:
```

Comment:
Again, there is already "yolo" alias. No need to duplicate at for every model.

Reply:
removed